internal/build/pom.xml

```diff
@@ -46,7 +46,6 @@
         <dep.asciidoctorj-diagram-ditaamini.version>1.0.3</dep.asciidoctorj-diagram-ditaamini.version>
         <dep.asciidoctorj-diagram.version>3.0.1</dep.asciidoctorj-diagram.version>
         <dep.asciidoctorj.version>3.0.0</dep.asciidoctorj.version>
-        <!-- leave at 3.x for now, 4.0.0 needs Java 17 -->
```

Comment:
well, if we don't update to 4.x.x, leave the comment in place so we remember to do so.